view/KKanghh/src/types/totalData.ts

```diff
@@ -0,0 +1,8 @@
+import { weatherData } from "./weatherData";
+
+export interface totalData {
+  jejuData: weatherData[];
+  seoulData: weatherData[];
+  daejeonData: weatherData[];
+  gwangjuData: weatherData[];
```

Comment:
[제안] 데이터 추가될때마다 타입늘어날 것 같은데 다음과 같은 형태는 어떠신가요?
```suggestion
  [dataName: string]: weatherData[];
```